build.sbt

```diff
@@ -3903,9 +3903,9 @@ lazy val `engine-runner` = project
               // by disabling this service provider
               "-H:ServiceLoaderFeatureExcludeServiceProviders=net.snowflake.client.core.FileTypeDetector",
               // useful perf & debug switches:
-              // "-g",
-              // "-H:+SourceLevelDebug",
-              // "-H:-DeleteLocalSymbols",
+              "-g",
+              "-H:+SourceLevelDebug",
+              "-H:-DeleteLocalSymbols",
```

Comment:
These flags should likely be controlled via `ENSO_LAUNCHER=native,debug` 
- in addition to https://github.com/enso-org/enso/blob/28b2e0d2f150ba8ff3445e953066a19be38be533/build.sbt#L5429